libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -529,6 +530,29 @@ bool AP_Arming::hardware_safety_check(bool report)
     return true;
 }
 
+bool AP_Arming::rc_arm_checks(AP_Arming::Method method)
+{
+    bool check_passed = true;
+    const RCMapper * rcmap = AP::rcmap();
+    if (rcmap != nullptr) {
+        const char *names[4] = {"Roll", "Pitch", "Throttle", "Yaw"};
```

Comment:
we need to account for sprung throttles. Maybe best just to ignore throttle?

Reply:
Throttle is now opt in via `RC_OPTIONS`